docs/edge-authorizers/security-and-management.mdx

```diff
@@ -30,33 +30,33 @@ Once an edge authorizer is connected to the Aserto control plane:
 
 ## Configuration
 
-To use an edge authorizer, you'll first need to create an edge authorizer connection. Head to the Connections tab, and
+To use an Edge Authorizer, you'll first need to create an Edge Authorizer connection. Head to the Connections tab, and
 create a new connection by clicking the "Add connection" button. Select the "Edge Authorizer" provider and provide a
-name and a display name for your edge authorizer connection.
+name and a display name for your authorizer connection.
 
 <img src="/edge-authorizer-connection.png" />
 
-Click on the newly created connection to retrieve the edge authorizer connection id, which will be used to configure the edge authorizer.
+Click on the newly created connection to retrieve the Edge Authorizer connection id, which will be used to configure the Edge Authorizer.
 
 <img src="/edge-authorizer-connection-entry.png" />
 
 ### Sidecar
 
-The Aserto CLI can be used to configure the Aserto sidecar to use the control plane by adding an option to the `configure` sub-command.
+The Aserto CLI can be used to configure the Topaz instance to use the control plane by adding an option to the `configure` sub-command.
```

Comment:
The Aserto CLI can be used to configure a Topaz instance to use the Control Plane..."